### ISSUE TITLE
TypeError for integration from infinity to a positive value

### ISSUE THREAD

**Issue by MonteCarloMichael** [sympy/sympy#13536]
The following code
```
from sympy import *
r = Symbol('r', real=True, positive=True)
a = Symbol('a', real=True, positive=True)
Integral(1/r**2,(r,oo,a)).doit()
```
results in an error

```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-110-1bc7a5c21a7a> in <module>()
      2 r = Symbol('r', real=True, positive=True)
      3 a = Symbol('a', real=True, positive=True)
----> 4 Integral(1/r**2,(r,oo,a)).doit()

~/anaconda3/lib/python3.6/site-packages/sympy/integrals/integrals.py in doit(self, **hints)
    464                 and not function.is_Poly and \
    465                     (xab[1].has(oo, -oo) or xab[2].has(oo, -oo)):
--> 466                 ret = try_meijerg(function, xab)
    467                 if ret is not None:
    468                     function = ret

~/anaconda3/lib/python3.6/site-packages/sympy/integrals/integrals.py in try_meijerg(function, xab)
    441                     x, a, b = xab
    442                     try:
--> 443                         res = meijerint_definite(function, x, a, b)
    444                     except NotImplementedError:
    445                         from sympy.integrals.meijerint import _debug

~/anaconda3/lib/python3.6/site-packages/sympy/integrals/meijerint.py in meijerint_definite(f, x, a, b)
   1801 
   1802     elif a == oo:
-> 1803         return -meijerint_definite(f, x, b, oo)
   1804 
   1805     elif (a, b) == (0, oo):

TypeError: bad operand type for unary -: 'tuple'
```

whereas

    -Integral(1/r**2,(r,a,oo)).doit()

, which is an equivalent formulation of the integral above, gives the correct result: `-1/a`. 

Furthermore, lowering the restrictions for the domain of `a`

    r = Symbol('r', real=True, positive=True)
    a = Symbol('a')
    Integral(1/r**2,(r,oo,a)).doit()
    

gives the correct result as well.

I posted this as a question on stackoverflow (https://stackoverflow.com/questions/46926912/sympy-integration-limits-error-typeerror-bad-operand-type-for-unary-tuple)  but the post was not recieved well. I want to point out again that it is completely valid to integrate from infinity to a positive value. I tested it with Wolfram Mathematica which gives the correct result (by using `Integrate[1/r^2, {r, Infinity, a}, Assumptions -> {Element[a,Reals], a > 0}]`).

**Comment by jksuom** [comment]
`meijerint_definite` returns a tuple that consists of the result of integration and the condition under which it is valid. Line 1803 should be replaced by `return -res[0], res[1]` where `res = meijerint_definite(f, x, b, oo)`.

**Comment by ashishkg0022** [comment]
@jksuom  may I send a pull request implementing the above mentioned changes ?

**Comment by jksuom** [comment]
@ashishkg0022 Yes, you can do that. And add a unit test as well.

**Comment by jksuom** [comment]
Closed by #13562
